src/devices/imuBosch_BNO055/imuBosch_BNO055.h

```diff
@@ -141,39 +142,46 @@ constexpr int MAX_MSG_LENGTH = 128;
 * | baudrate       | int    | Hz    |               | Yes if i2c not specified | baudrate setting of COM port | ex 115200, used only with serial configuration |
 * | i2c            | string |       |               | Yes if comport not specified | full name of device file  | ex '/dev/i2c-5', it is mutually exclusive with 'comport' parameter, necessary for i2c configuration|
 * | period         | int    | ms    |       10      | No       | period of the thread | |
+* | sensor_name    | string |       | bosch_imu_bno055 | No | full name of the device | |
+* | frame_name     | string |       | sensor_imu_bosch_bno055 | No | full name of the sensor frame in which the measurements are expressed | |
```

Comment:
Not sure what is the wisest thing to do w.r.t to this parameter: default values are always a bit dangerous, because they complexify debug  when something go wrong (for example, if you misspell `sensor_name` in the configuration file, if it is a required parameter you get an error, but if it has a default parameter then you silently get the default value. 
@barbalberto what is your opinion on this? A possibility is to print a deprecation warning if the parameters  are not set, and then actually make the parameters compulsory after a minor release cycle.

Reply:
In other devices like `grabberDual` when a parameter is not specified and default one is used a warning is printed at the startup, saying "parameter x not set, using default(value)". Imo this could be the best compromise, because default parameters make easier to run devices, but as @traversaro said can hide some errors in configuration phase.

Reply:
I agree with @Nicogene . I would  suggest to keep the default `sensor_name` == `frame_name`, as we are doing to do that in the actual deployment on the robot.

Reply:
Fixed it.